config/default.yaml

```diff
@@ -7,6 +7,7 @@ security:
 idam:
   base_url: http://localhost:4501
   s2s_url: http://localhost:4502
-  service_key:
+  service_key: AAAAAAAAAAAAAAAA
   service_name: ccd_ps
 case_data_store_url: http://localhost:4452
+case_data_probate_template_url: http://localhost:4104
```

Comment:
This is missing a matching entry in `config/custom-environment-variables.yaml` to allow for overriding that property per environment.

Reply:
Cool - that's exactly the kind of thing I thought you'd illuminate. I will add it there too